File: milestone 2/python_modules/module00/ex7/ft_seed_inventory.py
def ft_seed_inventory(seed_type: str, quantity: int, unit: str):
    seed_type = seed_type.capitalize()
    if unit == "packets":
        print(f"{seed_type} seeds: {quantity} {unit} available")
    elif unit == "grams":
        print(f"{seed_type} seeds: {quantity} {unit} total")
    elif unit == "area":
        unit = "square meters"
        print(f"{seed_type} seeds: covers {quantity} {unit}")
    else:
        print("Unknown unit type")


ft_seed_inventory("grams", 12, "")

File: milestone 2/python_modules/module00/ex7/ft_seed_inventory.cpp
#include <iostream>
#include <string>

void	ft_seed_inventory(std::string seed_type, int quantity, std::string unit)
{
	auto it = seed_type.begin();
	*it = static_cast<char>(std::toupper(*it));
	if (unit == "packets")
		std::cout << seed_type << " seeds: " << quantity << " " << unit << " " << "available";
	else if (unit == "grams")
		std::cout << seed_type << " seeds: " << quantity << " " << unit << " " << "total";
	else if (unit == "area")
	{
		unit = "square meters";
		std::cout << seed_type << " seeds: " << "covers" << " " << quantity << " " << unit;
	}
	else
		std::cout << "Unknown unit type";
	std::cout << std::endl;
	}

int main(void)
{
	ft_seed_inventory("tomato", 15, "packets");
	ft_seed_inventory("carrot", 8, "grams");
	ft_seed_inventory("lettuce", 12, "area");
	return 0;
}